src/components/DropdownMenu/DropdownMenuItem.tsx

```diff
@@ -100,7 +100,9 @@ export const DropdownMenuItem = <T,>({
                 onClick={handleMenuItemClick}
             >
                 <div className={cnDropdownMenu('menu-item-content')}>
-                    <div>{text || children}</div>
+                    <div className={cnDropdownMenu('menu-item-content-children')}>
```

Comment:
without this class name `div` will have width equal `text || children` width.
this is unacceptable if we want all items in DropDown have same width.

for exemple 

<img width="388" alt="image" src="https://github.com/gravity-ui/uikit/assets/43282255/fdb399cd-4739-4702-8a9d-3b28eb604b47">